api/rs/slint/tests/show_strongref.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+use ::slint::slint;
+
+// Sorry, can't test with rust test harness and multiple threads.
+#[cfg(not(any(target_arch = "wasm32", target_os = "macos", target_os = "ios")))]
+#[test]
+fn show_maintains_strong_reference() {
+    slint::platform::set_platform(Box::new(i_slint_backend_winit::Backend::new())).unwrap();
+
+    slint!(export component TestWindow inherits Window {
+        callback root-clicked();
+        TouchArea {
+            clicked => { root.root-clicked(); }
+        }
+    });
+
+    let window = TestWindow::new().unwrap();
+
+    let window_weak = window.as_weak();
+    let window_weak_2 = window_weak.clone();
+
+    slint::Timer::single_shot(std::time::Duration::from_millis(20), move || {
```

Comment:
do we need to wait 20ms  or can we just call invoke_in_event_loop and do it in the next iteration?

Reply:
`invoke_from_event_loop()` would be very nice to use, but it's not possible to call it before spinning the event loop (we don't have a queue) - instead `EventLoopError::NoEventLoopProvider` is returned as error.

But you're right about the timeout, I guess I can make that quicker :)

Reply:
really?  it should normally work as soon as the backend is set

Reply:
You're right. I misremembered (and when I ran into this myself the last time, there was indeed no backend yet).